src/rpc/rpc.cc

```diff
@@ -1190,10 +1190,11 @@ future<> server::connection::send_unknown_verb_reply(std::optional<rpc_clock_typ
   }
 
   future<> server::stop() {
+      _stop_watchdog.arm(std::chrono::minutes(1));
       return when_all(
           shutdown(),
```

Comment:
Maybe we should rearm `_stop_watchdog` in `shutdown()` after closing each connection so the watchdog deadline can be extended while there forward progress?

Also, the timeout can be longer (e.g. 5 minutes) to cover debug mode.